src/pages/dbot/index.js

```diff
@@ -1,28 +1,121 @@
 import React, { Component } from 'react'
 import styled from 'styled-components'
-import { Tutorial } from './_tutorial'
-import { TradeWithDbot } from './_trade-with-dbot'
-import AutomateTrading from './_automate-trading'
-import Banner from './_banner'
-import HowItWorks from './_how-it-works'
+import DBotAnimation from './dbot-animation.json'
```

Comment:
please add the lotties in `src/images/lotties` folder - the dbot, dtrader, dmt5 lotties will be used on the homepage too

Reply:
done